src/test/gen-ts_test.ts

```diff
@@ -21,11 +22,21 @@ const goldens = path.join(__dirname, '..', '..', 'src', 'test', 'goldens');
 suite('generateDeclarations', () => {
   for (const fixture of fs.readdirSync(goldens)) {
     test(fixture, async () => {
-      const golden =
-          fs.readFileSync(path.join(goldens, fixture, 'expected.d.ts'));
-      const declarations =
-          await generateDeclarations(path.join(fixtures, fixture));
-      assert.equal(declarations, golden.toString());
+      // TODO Make these tests more async.
```

Comment:
More async in what way?

Reply:
Analyzing and reading all the golden files is done synchronously. They could be done concurrently. The bottleneck by far is analyzer though, so I'm not bothered right now, as it makes the code harder to read. Moved the TODO and made it more clear.